services/workflows-service/src/workflow/utils/toPrismaWhere.ts

```diff
@@ -0,0 +1,36 @@
+import { Prisma, WorkflowRuntimeDataStatus } from '@prisma/client';
+
+type Filters = {
+  assigneeId?: (string | null)[];
+  status?: WorkflowRuntimeDataStatus[];
+};
+
+export const toPrismaWhere = (filters: Filters): Prisma.WorkflowRuntimeDataWhereInput => {
```

Comment:
This function, and the `toPrismaOrderBy`, are workflow-specific. They won't fit for users, etc.
Should I rename them or place them in another folder to make it clearer?